### ISSUE TITLE
Not Detecting the Card

### ISSUE THREAD

**Issue by sriharshamvs** [arozcan/MFRC522-I2C-Library#2]
Hi,
I have uploaded the same code to NodeMCU and got This error
![error](https://user-images.githubusercontent.com/31341894/31018319-2646c11e-a51b-11e7-9a25-97aec9a27e34.png)

The board is perfectly working with SPI protocol. No Problem with the board.

**Comment by paulmassen** [comment]
Same problem here!

**Comment by capedra** [comment]
@sriharshamvs @paulmassen @arozcan 
I also have the same problem. Maybe it's because even though the Mifare chip supports SPI, I2C, and UART, if you’re using the most common breakout board (RFID-RC522), it’s hard wired to only support SPI.

**Comment by thr33bricks** [comment]
Yes, most of these boards only support **SPI** unless you cut some connections. Here is a thread on how to **turn on** the **I2C mode**. https://forum.arduino.cc/index.php?topic=442750.0
